### PR TITLE
Feature/custom push

### PR DESCRIPTION
**Pull request by mistikk**
### Screenshots/Video
image

### CODE REVIEW

**Comment by horpeyloaded** [comment]
Please what about **Transfer** @mistikk

**Comment by mistikk** [comment]
It is bottom of reblog but it is not showing on ss @horpeyloaded 
image

**Comment by ue** [comment]
@mistikk conflit

**Comment by mistikk** [comment]
@ue fixed

**Comment by ue** [comment]
@mistikk conflit

**Comment by ue** [comment]
maybe its not a good way for puting all together to redux store we can get directly from realm when it required I know we put that to redux store I was throuth it will be quick ones but now its goes big one what u thing?

**Comment by ue** [comment]
@mistikk  confilt

**Comment by mistikk** [comment]
@ue fixed

File: src/screens/application/container/applicationContainer.js [hunk 1]
```diff
@@ -251,7 +246,14 @@ class ApplicationContainer extends Component {
...
         if (response.isDefaultFooter !== '') dispatch(isDefaultFooter(response.isDefaultFooter));
         if (response.notification !== '') {
-          dispatch(isNotificationOpen(response.notification));
+          dispatch(changeNotificationSettings({ type: 'notification', action: response.notification }));
+          dispatch(changeNotificationSettings({ type: 'notification.follow', action: response.followNotification }));
+          dispatch(changeNotificationSettings({ type: 'notification.vote', action: response.voteNotification }));
+          dispatch(changeNotificationSettings({ type: 'notification.comment', action: response.commentNotification }));
+          dispatch(changeNotificationSettings({ type: 'notification.mention', action: response.mentionNotification }));
+          dispatch(changeNotificationSettings({ type: 'notification.reblog', action: response.reblogNotification }));
+          dispatch(changeNotificationSettings({ type: 'notification.transfers', action: response.transfersNotification }));
```
Comment (ue) [inline src/screens/application/container/applicationContainer.js]: refactoring need and what kind of dispatch is that why you need that?

**Comment by ue** [comment]
Screen Shot 2019-03-24 at 23 08 21

**Comment by mistikk** [comment]
@ue fixed

**Pull request merged**